packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -184,6 +184,16 @@
                       "gap_duration_s": {
                         "type": "float"
                       },
+                      "gap_range": {
```

Comment:
Just wanted to make sure, I may have missed it, are we actually searching over these dates? If not we do not need mappings for them.

Reply:
@nkhristinin are we planning to query directly over the rule SO for gaps?

Reply:
Hm, don't think so, I till remove it